tests/test_tasks.py

```diff
@@ -12,6 +19,45 @@
 has_celery = celery is not None
 
 
+@pytest.mark.django_db
+def test_cleanup_request_logs_command(settings, caplog):
+    settings.LOG_OUTGOING_REQUESTS_MAX_AGE = 1  # delete if > 1 day old
+
+    with freeze_time("2023-10-02T12:00:00Z") as frozen_time:
+        OutgoingRequestsLog.objects.create(timestamp=timezone.now())
+        frozen_time.move_to("2023-10-04T12:00:00Z")
+        recent_log = OutgoingRequestsLog.objects.create(timestamp=timezone.now())
+
+        with caplog.at_level(logging.INFO):
+            call_command("prune_logs", stdout=StringIO(), stderr=StringIO())
+
+    assert len(caplog.records) == 1
+    assert "Deleted 1 outgoing request log(s)" in caplog.text
```

Comment:
this should check actual command output:

```suggestion
    stdout = StringIO()
        with caplog.at_level(logging.INFO):
            call_command("prune_logs", stdout=stdout, stderr=StringIO())

    output = stdout.getvalue()
    assert output == "Deleted 1 outgoing request log(s)"
```